### ISSUE TITLE
Lua syntax highlighting is broken

### ISSUE THREAD

**Issue by appgurueu** [scopatz/nanorc#393]
See i.e. https://github.com/zyedidia/micro/pull/2426 and https://github.com/zyedidia/micro/pull/2409

**Comment by davidhcefx** [comment]
Wait that's a different text editor. Are you sure you reported at the correct place?

**Comment by appgurueu** [comment]
> Wait that's a different text editor. Are you sure you reported at the correct place?

Yes. Sorry, I should have elaborated. Micro's syntax highlighting rules seem to be derived from this repo. I looked at the files in this repo, and they seem to have the same issues micro seems to have inherited from here. The two linked issues should hence apply to this repo all the same.
